docs/changelog/84323.yaml

```diff
@@ -0,0 +1,5 @@
+pr: 84323
+summary: Small cleanup to help generate JWT realm from static config settings
+area: Authentication
+type: feature
```

Comment:
This PR should be labelled `>non-issue` rather than `>feature`.

This PR doesn't introduce a brand new feature.
Since the changes it makes are too an unreleased feature, the PR itself is `>non-issue` because people upgrading from an earlier version of ES to the version that contains this PR don't need to know about this specific PR, just the overall PR that introduced JWT.

Reply:
Changed to >non-issue. Resolved.